README.md

```diff
@@ -242,4 +242,18 @@ to be used in production. The next steps we plan to work on are:
     forbidden, making it not Turing-complete.
 
 See [RATIONALE.md](./RATIONALE.md) for the design rationale and a more detailed
-comparison with a selection of these languages.
+comparison with these languages.
+
+### Summary
+
+| Language | Typing                        | Recursion  | Evaluation | Side-effects                                     |
+|----------|-------------------------------|------------|------------|--------------------------------------------------|
+| Nickel   | Gradual (dynamic + static)    | Yes        | Lazy       | Yes (constrained)                                |
+| Starlark | Dynamic                       | No         | Strict     | No                                               |
+| Nix      | Dynamic                       | Yes        | Lazy       | Predefined and specialized to package management |
+| Dhall    | Static (requires annotations) | Restricted | Lazy       | No                                               |
+| CUE      | Static (everything is a type) | No         | Lazy       | No, but allowed in the separated scripting layer |
+| Jsonnet  | Dynamic                       | Yes        | Lazy       | No                                               |
+| JSON     | None                          | No         | Strict     | No                                               |
```

Comment:
I don't think talking about strict evaluation for data language makes a lot of sense. What about something like `N/A`?

Reply:
Done in 13a35021dfab9e7d0436e4245dca41935d1f19c3